server/api/utils/helpers.py

```diff
@@ -119,10 +119,10 @@ def is_request_from_leader(
     return False
 
 
-def string_to_timedelta(date_str, raise_on_error=True):
+def string_to_timedelta(date_str, interval=0, raise_on_error=True):
```

Comment:
```suggestion
def string_to_timedelta(date_str, offset=0, raise_on_error=True):
```
a more indicative name IMO